src/sswsdk/interface.py

```diff
@@ -134,10 +134,15 @@ class DBInterface(object):
 
     db_map = dict()
 
-    def __init__(self):
+    def __init__(self, **kwargs):
 
         super(DBInterface, self).__init__()
 
+        # Store the arguments for redis client
+        self.redis_kwargs = kwargs
+        if len(self.redis_kwargs) == 0:
+            self.redis_kwargs['unix_socket_path'] = self.REDIS_UNIX_SOCKET_PATH
+
```

Comment:
I will not change it if the kwargs is non empty.
The [redis client](https://redis-py.readthedocs.io/en/latest/#redis.StrictRedis) will smartly handle all the kwargs. If no unix_socket_path provided, it will open a network socket, either according to kwargs or default values